.github/workflows/code-analysis.yml

```
@@ -10,7 +10,31 @@ jobs:
   php-stan:
     runs-on: ubuntu-latest
     steps:
-    - name: Pull repo
-      uses: actions/checkout@v2
-    - run: composer install
-    - run: vendor/bin/phpstan analyze
+     
+     - name: Install wordpress
+       run: |
+         curl -O https://wordpress.org/latest.tar.gz
+         tar xzvf latest.tar.gz 
+         rm latest.tar.gz
+         pwd
+         ls -al
+     - name: Install woocommerce
+       run: |
+         curl -O https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip
+         unzip woocommerce.latest-stable.zip 
+         rm woocommerce.latest-stable.zip
+         pwd
+         ls -al
+       working-directory: /home/runner/work/TestRepo/TestRepo/wordpress/wp-content/plugins
```

Can we use ~ so that the home path isn't hardcoded?
```suggestion
       working-directory: ~/work/TestRepo/TestRepo/wordpress/wp-content/plugins
```